-------------------------------------------------------------------------------
Test set: de.vsy.chat.server.testing_grounds.TestServerSocketIO
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0 s <<< FAILURE! - in de.vsy.chat.server.testing_grounds.TestServerSocketIO
TestServerSocketCloseIOException  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.io.IOException to be thrown, but nothing was thrown.
	at de.vsy.chat.server.testing_grounds.TestServerSocketIO.TestServerSocketCloseIOException(ServerSocketIO.java:18)

